content/php/concepts/arrays/terms/array-fill/array-fill.md

```diff
@@ -0,0 +1,64 @@
+---
+Title: 'array_fill()'
+Description: 'This function fills an array with values and returns a filled array.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+  - 'Arrays'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`array_fill()`** method is used to fill an array with values. This function creates a custom array with a given pre-filled value.
+
+## Syntax
+
+```pseudo
+array_fill($start_index, $number_elements, $values)
+```
+
+The `array_fill()` function has three required parameters:
+
+- `$start_index`: This parameter specifies the first `index` of the returned array.
```

Comment:
```suggestion
- `$start_index`: This parameter specifies the first index of the returned array.
```

Reply:
generally elements that are being referenced in a code block such as a variable name or function should use backticks. Generic terms for elements do not have to be formatted.